tests/fixtures/select_one_legacy/v1.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": "romev1",
+  "content": {
+    "survey": [
+      {
+        "type": "select one",
```

Comment:
I'm a little confused how it gets to this position.
The legacy format is:

`type: "select one emperors"`

then the form is "standardized" and it should be changed to:

```
type: "select_one"
select_from_list_name: "emperors"
```

I don't doubt that this is a case that occurs in existing forms, but would it make sense to expand the test to cover the way it was uploaded from those legacy xlsforms?

Reply:
Oh ok i see now that in this commit #3c12f05 , we've changed the _expand_type_to_dict to allow a lot of alternatives.

I think this would be a good place to change `select one` to `select_one`. And then (I think) we could avoid adding `select[space]one` as an alias in fields.py.